typings/index.d.ts

```diff
@@ -4475,12 +4478,15 @@ export type GuildFeatures =
   | 'PRIVATE_THREADS'
   | 'ROLE_ICONS';
 
+export type TimeoutDateResolvable = DateResolvable;
```

Comment:
What's the point of this type?

Reply:
The same reason as other types that do this, I suppose. Future-proofing if the types need to be amended and also to match with the relevant JSDoc.

Reply:
I don’t recall seeing many other types that do this and consistency with JSDoc has never been a rule to my knowledge. I think this type could be easily removed